outlines/publish-packages.md

```diff
@@ -0,0 +1,27 @@
+# Publishing a Package
+
+1. Pros/cons of publishing a package
+  1. Considering what's involved in publishing a package.
+  2. Do you think you'll be able to maintain it properly?
+  3. Is the code complete enough to be useful, or does it need work to be made generic?
+2. How to decide what's in a package / more than one
+  1. Is there any core functionality that could be reused elsewhere? (e.g. route matching)
+  2. Is there pure JS code that could be published to NPM?
+  3. Principle of a single symbol per package could help (see structure chapter).
+3. How to write great documentation.
+  1. README vs longer documentation/guide
+  2. Is the documentation too complicated? Maybe the package should be simpler?
+  3. Writing guide documentation ;)
```

Comment:
Hmm, I feel like people shouldn't be building packages with the explicit intent of getting them into the guide - that process should include first becoming a de-facto standard package IMO? Perhaps this is "if you think your package should be included in the guide, here's what to do"